### ISSUE TITLE
`aumix` fails to evaluate 

### ISSUE THREAD

**Issue by deepfire** [NixOS/nixpkgs#22663]
## Issue description
```
[nix-shell:~]$ nix-env -i aumix
error: attribute ‘features’ missing, at /home/deepfire/src/nixpkgs/pkgs/os-specific/linux/phc-intel/default.nix:7:9
```
## Technical details

* System: NixOS `master`, as of Feb 10 2017
* Nix version: `1.11.6`
* Nixpkgs version: `17.03.git.ca8fb930b1`

**Comment by dezgeg** [comment]
I think it's fixed after fc54ece4709d39a9feaa61600d23a0490ce49bd1.

**Comment by pSub** [comment]
This issue is fixed on 17.09 and unstable.
